localstack/services/stepfunctions/stepfunctions_utils.py

```diff
@@ -18,3 +19,14 @@ def _get_result():
         return result
 
     return retry(_get_result, sleep=2, retries=timeout_secs / 2)
+
+
+def is_old_provider():
```

Comment:
nit: we could probably move this to the test files, as this check seems to be only used in the integration tests..? (not in the provider implementation itself)